tests/fixtures/v1/manualSend.html

```diff
@@ -2,12 +2,12 @@
 <html>
   <head>
     <title>Raygun4JS with V1 API</title>
-    <script src="/fixtures/common/instrumentXHRs.js"></script>
+    <script src="./../common/instrumentXHRs.js"></script>
     <script src="/dist/raygun.js"></script>
   </head>
   <body>
     <script type="text/javascript">
-      Raygun.init('abcdef==', {allowInsecureSubmissions: true});
```

Comment:
How come we needed to disable `allowInsecureSubmissions` ?
will this lead to some headaches down the line if the test runs it over HTTP?

Reply:
I've noticed this approach on the other files - so if changes are needed here then we should also remember to update the other files